Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.Source.cs

```diff
@@ -106,6 +106,14 @@ private static bool IsHttpUri(Uri uri)
                 if (IsCacheEnabled && _isHttpSource)
                 {
                     _image.Source = await ImageCache.GetFromCacheAsync(_uri);
+                    if (_image.Source != null)
```

Comment:
Loaded event on _image never gets fired when source BitmapImage is not created from Uri, but instead it's source is set with SetSource(stream), which is the case now because memory cache was implemented in ImageCache. Maybe there is more elegant solution.